package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "dataset-addon-for-splunk",
+  "version": "1.0.0",
```

Comment:
I assume that right now this is only used for managing test dependencies aka playwright, right?

If so, it would be good to clarify this in the readme and as part of this file description.

Reply:
If so, we should also add ``private: true`` attribute (https://stackoverflow.com/questions/15711709/can-i-have-a-package-json-but-avoid-my-project-from-getting-published-to-npm-ser) to indicate this this package is purely used for managing dependencies and not actually used for creating an artifact we publish to npm or similar.

Reply:
Fixed